apps/docs/build.md

```diff
@@ -0,0 +1,218 @@
+# Apps Build System
+
+The apps build system generates all the client side javascript files used on
+studio.code.org. This file contains documentation about how it works and how it
+can be configured for different needs.
```

Comment:
This is accurate, however it makes me wish this was called something besides (or more specific than) "apps".